.github/workflows/ci.yml

```diff
@@ -81,4 +86,34 @@ jobs:
         if: always()
         with:
           name: reports
-          path: jbehave-support-core/target/reports
\ No newline at end of file
+          path: jbehave-support-core/target/reports
+      - name: Upload test result for integration tests
+        uses: actions/upload-artifact@v1
+        with:
+          name: jacoco-it
+          path: jbehave-support-core/target/site/jacoco-it/jacoco.xml
+
+  sonar:
+    name: Sonar integration
+    needs: [ build, test, system-test ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Download result for test execution
+        uses: actions/download-artifact@v1
+        with:
+          name: jacoco
+      - name: Download result for integration test execution
+        uses: actions/download-artifact@v1
+        with:
+          name: jacoco-it
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Run Maven
+        run: mvn package org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=xsmrcek_jbehave-support -Dsonar.organization=xsmrcek -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=${{ secrets.SONAR_TOKEN }} -Dsonar.coverage.jacoco.xmlReportPaths=/home/runner/work/jbehave-support/jbehave-support/jacoco/jacoco.xml,/home/runner/work/jbehave-support/jbehave-support/jacoco-it/jacoco.xml
```

Comment:
Just a question do we have to have absolute path to the jacoco files?
And if so can we use something like `pwd` or something like that for the first part of the path? (I am just afraid that the path on the github runners can change in the future)

Reply:
I had to. But well we may figurer out how to use shorter version :)

Reply:
Ok, thanks 🙂 One other thing - I would add skipTests to this command so it does not run unit tests (or maybe `mvn compile` would be enough instead of `mvn package`?) But thats just nitpicking 🙂